### DISCUSSION TITLE
Is it possible to enable UART Terminal?

### DISCUSSION THREAD

**Discussion by kahlenberg** [raspiblitz/raspiblitz#4237] [Q&A] [answered]
I want to connect an USB-Uart converter to Raspberry Pi and want to be able to log in into serial port terminal.
I already tried to activate it  in /boot/config.txt with

`dtoverlay=disable-bt
enable_uart=1 
`
and in cmdline.txt
`console=serial0,115200 console=tty0`

but nothing is sent over UART.
Is it possible to activate the UART terminal and what is the proper way to activate it in raspiblitz?
Thanks.

**Comment by openoms** [comment, marked as answer]
You can eneble the serial port with:
```
sudo raspi-config
```
Interface Options -> Serial Port
